owsi-core/owsi-core-components/owsi-core-component-jpa/src/main/java/fr/openwide/core/jpa/query/AbstractQuery.java

```diff
@@ -0,0 +1,25 @@
+package fr.openwide.core.jpa.query;
+
+import java.util.List;
+
+public class AbstractQuery<T, TQueryImpl> implements IQuery<T> {
+
+	@Override
+	public List<T> fullList() {
+		// TODO Auto-generated method stub
```

Comment:
Quite sure it should not ^^ I'll take a look, but this file should probably be deleted.